Berlin/BerOrOct555.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="BerOrOct555">
+   <teiHeader xml:base="https://betamasaheft.eu/">
+      <fileDesc>
+         <titleStmt>
+            <title xml:id="t1"> Gadla Gabra Manfas Qǝddus, Taʾammǝra Gabra Manfas Qǝddus </title>
+            <editor key="SD"/>
+         </titleStmt>
+         <editionStmt>
+            <p>Cataloguing funded by the AHRC-DFG Project
+               'Demarginalizing medieval Africa: Images, texts, and identity
+               in early Solomonic Ethiopia (1270-1527)', grant ref.
+               nos. AH/V002910/1 - 448410109 <ptr target="https://gepris.dfg.de/gepris/projekt/448410109"/>.</p>
+         </editionStmt>
+         <publicationStmt>
+            <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+            <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+            <pubPlace>Hamburg</pubPlace>
+            <availability>
+               <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                  <p> This file is licensed under the Creative Commons
+                                            Attribution-ShareAlike 4.0. </p>
+               </licence>
+            </availability>
+         </publicationStmt>
+
+         <sourceDesc>
+            <msDesc xml:id="ms">
+               <msIdentifier>
+                  <repository ref="INS0333SBB"/>
+                  <collection>Ms. orient. oct.</collection>
+
+                  <idno facs="https://content.staatsbibliothek-berlin.de/dc/863181112/manifest">SBP, Ms. orient. oct. 555</idno>
+               </msIdentifier>
+
+               <additional>
+                  <adminInfo>
+                  <recordHist>
+                  <source>
+                    <listBibl type="secondary">
+                    <bibl>
+                    <ptr target="bm:HammerschmidtJaeger1968Illuminiert"/>
+                      <citedRange unit="page">50-54</citedRange>
+                    </bibl>
+                    </listBibl>
+                    </source>
+                  </recordHist>
+                  </adminInfo>
+                </additional>
+
+            <msPart xml:id="p1">
+               <msIdentifier> <idno>SBP, Ms. orient. oct. 555, p1</idno>
```

Comment:
this idno is not necessary, I don't think that p1 will be understandable to those who don't encode